packages/ui/src/accounts/components/AccountItem/components/LocksDetails.tsx

```diff
@@ -2,8 +2,7 @@ import React from 'react'
 
 import { LockItemWrapper } from '@/accounts/components/AccountItem/components/LockItemWrapper'
 import { VestingLockListItem } from '@/accounts/components/AccountItem/components/VestingLockItem/VestingLockItem'
-import { useRecoveryConditions } from '@/accounts/hooks/useRecoveryConditions'
-import { isRecoverable } from '@/accounts/model/lockTypes'
+import { useGroupLocks } from '@/accounts/hooks/useGroupLocks/useGroupLocks'
```

Comment:
For consistency with the project please add a `useGroupLocks/index.ts` so that:
```suggestion
import { useGroupLocks } from '@/accounts/hooks/useGroupLocks'
```